client/js/components/procedure/admin/DpAddOrganisationList.vue

```diff
@@ -31,22 +31,22 @@
               {{ Translator.trans('address') }}
             </dt>
             <dd
-              v-if="locationContacts.street"
+              v-if="locationContacts?.street"
```

Comment:
If `locationContacts` is not defined, wouldn't that mean that only the text "address" is displayed, but no address data? So maybe we could add a `v-if="locationContacts"` to the `dl` above, which should also make adding all the `?` below unnecessary?

Reply:
No, if there is no adress data, the translation `Translator.trans('notspecified')` is shown. But changed it a bit here to make it more clear: [23f1f9b](https://github.com/demos-europe/demosplan-core/pull/3649/commits/23f1f9bbf351eaf5ada2a7c69d9a9ce530170ab7)

Reply:
And here: [beba1ae](https://github.com/demos-europe/demosplan-core/pull/3649/commits/beba1ae0399c5261458267769c03810d3646a2a2) forgot the check in the method earlier